t/08-autotest.t

```diff
@@ -97,7 +97,7 @@ loadtest 'start', 'rescheduling same step later';
 is(keys %autotest::tests, 3, 'three steps have been scheduled (one twice)') || diag explain %autotest::tests;
 is($autotest::tests{'tests-start1'}->{name}, 'start#1', 'handle duplicate tests');
 is($autotest::tests{'tests-start1'}->{$_}, $autotest::tests{'tests-start'}->{$_}, "duplicate tests point to the same $_")
-  for qw(script fullname category class);
+  for qw(script category class);
```

Comment:
I suggest to add a new test for fullname though, instead of just leaving it out.